_posts/2018-11-12-writing-fast-rails-part-2.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "Tips for Writing Fast Rails: Part 2"
+date: 2018-11-12 12:10:00
+categories: ["rails", "performance"]
+author: "luciano"
+---
+
+Some time ago we wrote a couple of [Tips for Writing Fast Rails](https://www.ombulabs.com/blog/performance/rails/writing-fast-rails.html). Now we decided to continue sharing our knowledge on this topic so here is the part two!
```

Comment:
> Now we decided to continue sharing our knowledge on this topic so here is the part two!

It could sound more natural like this: 

> It was about time we wrote part two so here it is!